corehq/apps/translations/app_translations/utils.py

```diff
@@ -73,11 +73,8 @@ def get_bulk_app_sheet_headers(app, lang=None, exclude_module=None, exclude_form
             if exclude_form is not None and exclude_form(form):
                 continue
 
-            sheet_name = get_form_sheet_name(form)
-            headers.append([
-                sheet_name,
-                ["label"] + lang_list
-            ])
+            sheet_name = form.unique_id if by_id else get_form_sheet_name(form)
```

Comment:
`sheet_name` should now be `sheet_identifier` or `sheet_name_or_unique_id`?
Would also need change at few places and in the documentation. :(